modules/static_date.js

```diff
@@ -0,0 +1,7 @@
+function actualTime() {
```

Comment:
- [ ] As required by the first reviewer, Please use arrow syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html)